synapse/lib/layer.py

```diff
@@ -3398,6 +3484,13 @@ async def getModelVers(self):
         return self.layrinfo.get('model:version', (-1, -1, -1))
 
     async def setModelVers(self, vers):
+        await self._push('layer:set:modelvers', vers)
+
+    @s_nexus.Pusher.onPush('layer:set:modelvers')
+    async def _setModelVers(self, vers):
+        if vers == (0, 2, 7):
```

Comment:
i don't think so since the only case where this actually changes the stortype is in a mirror which is about to be migrated. if the mirror was using the old stortype and got a version > (0, 2, 7) it would have somehow missed the migration